package.json

```diff
@@ -26,17 +25,27 @@
     "webpack-dev-server": "^1.14.1"
   },
   "dependencies": {
-    "contentful": "^3.1.1",
+    "axios": "^0.12.0",
+    "contentful": "^3.5.0",
     "flux": "^2.1.1",
+    "history": "^2.0.1",
     "localforage": "^1.4.0",
     "marked": "^0.3.5",
     "pretty-bytes": "^3.0.1",
     "react": "^15.0.0-rc.1",
+    "react-ace": "^3.4.1",
     "react-addons-update": "^15.1.0",
     "react-css-modules": "^3.7.5",
     "react-dom": "^15.0.0-rc.1",
+    "react-json-tree": "^0.8.0",
+    "react-redux": "^4.4.5",
     "react-router": "^2.0.0",
-    "history": "^2.0.1",
+    "react-router-redux": "^4.0.5",
+    "redux": "^3.5.2",
+    "redux-logger": "^2.6.1",
+    "redux-promise": "^0.5.3",
+    "redux-promise-middleware": "^3.3.0",
+    "redux-thunk": "^2.1.0",
     "scourjs": "^1.0.1",
     "unique-concat": "^0.2.2"
   },
```

Comment:
Just an idea - should we add a `shrinkwrap` file, to guarantee the same deps for everybody?